src/audio/buffer.c

```diff
@@ -185,7 +185,8 @@ void comp_update_buffer_produce(struct comp_buffer *buffer, uint32_t bytes)
 	addr = buffer->stream.addr;
 
```

Comment:
Add prefix 'treewide' to commit subject. e.g: ```Remove references to .free and .avail members of audio_stream. ```

Reply:
done